res/css/views/voip/_CallView.scss

```diff
@@ -112,6 +112,7 @@ limitations under the License.
     z-index: 30;
     border-radius: 8px;
     overflow: hidden;
+    display: flex;
```

Comment:
was it not possible to split this out? seems like the layout code should be independent

Reply:
I am afraid I don't understand - sorry...

Reply:
Oh, I think I've realized what you mean